### PR TITLE
Android

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/2969

### CODE REVIEW

**Comment by mit-mit** [comment]
Sorry, a bit hard to see what the changes are. Did you just re-break the lines?

**Comment by sfshaza2** [comment]
@mit-mit, here is the change that's relevant to the bug:

  * The `-storetype JKS` tag is only required for Java 9
    or newer. As of this release, the keystore type defaults
    to PKS12.

Plus, the -storetype JKS tag was added to the command.

**Comment by mit-mit** [comment]
Oh, sorry I don't know anything about that.

File: src/docs/deployment/android.md [hunk 1]
```diff
@@ -60,29 +62,36 @@ If you have an existing keystore, skip to the next step.
...
-  part of the Java JDK, which is installed as part of Android Studio.
-  For the concrete path,
-  run `flutter doctor -v` and locate the path printed after
-  'Java binary at:'. Then use that fully qualified path
-  replacing `java` (at the end) with `keytool`.
-  If your path includes space-separated names,
-  such as `Program Files`, place quotes around the space-separated
-  names. For example: `/"Program Files"/`
+  * The `keytool` command might not be in your path&mdash;it's
+    part of the Java JDK, which is installed as part of
```
Comment (kwalrath) [inline src/docs/deployment/android.md]: Is "Java JDK" redundant?

File: src/docs/deployment/android.md [hunk 2]
```diff
@@ -60,29 +62,36 @@ If you have an existing keystore, skip to the next step.
...
+    Android Studio.  For the concrete path,
+    run `flutter doctor -v` and locate the path printed after
+    'Java binary at:'. Then use that fully qualified path
+    replacing `java` (at the end) with `keytool`.
+    If your path includes space-separated names,
+    such as `Program Files`, place quotes around the space-separated
+    names. For example: `/"Program Files"/`
+
+  * The `-storetype JKS` tag is only required for Java 9
+    or newer. As of this release, the keystore type defaults
```
Comment (kwalrath) [inline src/docs/deployment/android.md]: What does "this release" refer to? Java 9? Or a Flutter release? (I don't understand the tech; you might want to get someone who does understand to check your words.)

**Comment by sfshaza2** [comment]
Yes, as of Java 9.

**Pull request merged**